src/modules/spaces/actions.js

```diff
@@ -52,11 +63,10 @@ export function fetchById(spaceId) {
       }
       else if (value) {
         dispatch(sActions.fetchSuccess([value]))
-
-        const users = getState().users
-        const user_id = value.user_id
-        const has_user = !!(users.find(e => e.id === user_id))
-        if (!has_user) { dispatch(userActions.fetchById(user_id)) }
+        // TODO(matthew): Right now, the space has an embedded user and organization record... why are we doing this
+        // extra fetching?
```

Comment:
it may well be unnecessary now, but we may want to merge this code before refactoring that.